modules/channel-teams/src/config.ts

```diff
@@ -0,0 +1,13 @@
+export interface Config {
+  /** The microsoft appId for this bot */
+  microsoftAppId: string
+
+  /** The microsoft app Password (secret) to use for this bot */
+  microsoftAppPassword: string
+
+  /** Enable or disable this channel for this bot */
+  enabled: boolean
+
+  // /** Force usage of webhooks */
+  // forceWebhook: boolean
```

Comment:
Unused configuration, should be removed. Also, it could be a good idea to set default values for the configurations, since they will be displayed in  the config file in the global folder. Users will only have to copy them then edit it

Reply:
I don't know what default values to use.